portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -20,13 +22,18 @@
         content: the comment.
         timestamp: System time user posted the comment
     */
```

Comment:
Add id: to the argument documentation, please.
Probably something like: unique_id from the datastore.